ts/Core/Chart/Chart.ts

```diff
@@ -477,6 +477,31 @@ class Chart {
             this.callback = callback;
             this.isResizing = 0;
 
+            const zooming = optionsChart.zooming = optionsChart.zooming || {};
+
+            if (zooming) {
```

Comment:
Unnecessary condition? As far as I can see, `zooming` should now always be truthy.

Reply:
True, this condition it's no longer needed.